indy_node/server/pool_req_handler.py

```diff
@@ -34,6 +38,13 @@ def authErrorWhileAddingNode(self, request):
         if error:
             return error
 
+        dest = operation.get(TARGET_NYM)
+        did = base58.b58decode(dest)
```

Comment:
Can we use some helper method from (like `ed25519PkToCurve25519`) for this?

Reply:
yes, will be fixed.